src/env.js

```diff
@@ -15,7 +15,7 @@ dotenv.config();
 
 module.exports = {
   email: process.env.GOOGLE_CLIENT_EMAIL,
-  key: process.env.GOOGLE_PRIVATE_KEY.replace(/\\n/g, '\n'),
+  key: process.env.GOOGLE_PRIVATE_KEY,
```

Comment:
Why do you remove the code you just added with https://github.com/adobe/helix-run-query/pull/31 ?

Reply:
It's the same code; I must have switched branches with the commit! I'll just delete the other PR